pkg/build/nodeimage/internal/container/docker/archive.go

```diff
@@ -202,7 +220,7 @@ func editManifestRepositories(raw []byte, editRepositories func(string) string)
 		for i, tag := range entry.RepoTags {
 			parts := strings.Split(tag, ":")
 			if len(parts) > 2 {
-				return nil, fmt.Errorf("invalid repotag: %s", entry)
+				return nil, fmt.Errorf("invalid repotag: %v", entry)
```

Comment:
(This is for silencing `go vet`: `fmt.Errorf format %s has arg entry of wrong type sigs.k8s.io/kind/pkg/build/nodeimage/internal/container/docker.metadataEntry`)